backend/daemon/api/accounts/v1alpha/accounts_test.go

```diff
@@ -29,8 +29,9 @@ func TestGetAccount_Own(t *testing.T) {
 				DeviceId: "12D3KooWFMTJanyH3XttUC2AmS9fZnbeYsxbAjSEvyCeHVbHBX3C",
 			},
 		},
+		IsTrusted: true,
 	}
-
+	time.Sleep(100 * time.Millisecond) // give time to trust own account
```

Comment:
@juligasa Why is this needed? Aren't we setting the trust for ourselves right when we register account?

Reply:
You're right. That was there bc the old way to trust ourselves, now it's immediate!